### DISCUSSION TITLE
Ask questions about new commits to a repo

### DISCUSSION THREAD

**Discussion by its-aye-lisa** [getdevgen/devgen#4] [General]
Hello, I'm trying to use devgen so that I can better understand the commits to an open source repo that I follow and use. I am a non-developer but software development education; so I can understand vaguely from the code changes; but I'm not acquianted enough to the code or have enough day to day software development experience to know what the changes are and it'd be nice if I could use devgen to get a summary of the changes. 

For example, a commit from last night is: [feat: position section expanded by default if any value is present
](https://github.com/plasmicapp/plasmic/commit/b64d6f38343f8163215c5dac3889fdc5fa89804d); as a user of the software, its hard for me to tell if that that change is for the left side bar for the token area or when inserting an element to the canvas and there is text in it, whether or if it changes the behavior of a component

**Comment by imotai** [comment]
@its-a-lisa 
Hello! Great to hear you’re using Devgen to track updates in the project.

The next version will support chat to the commit!
For now you can try chat to the pull request， Just right click on the pull request page.

**Comment by imotai** [comment]
@its-a-lisa
You can now try the latest version of Devgen by following this guide: https://devgen.xyz/docs/tutorial/chat_with_commit.

Let me know if you have any feedback or encounter any bugs!

**Comment by its-aye-lisa** [comment]
Can you slow that video down a little bit? Its hard to see what action was done after clicking on the commit and what was selected from the right click. 

Also, it seemed to work when asking a single question about a commit. But when I went to ask a follow up question, I got an error

![image](https://github.com/user-attachments/assets/f81beb3e-90c9-43b5-b927-fa27c6cc74eb)

**Reply by imotai** [reply]
I will improve the demo video and check the error log. Thank you for the feedback.
